proxy/src/control/destination/background.rs

```diff
@@ -70,6 +72,7 @@ struct DestinationSet<T: HttpService<ResponseBody = RecvBody>> {
     query: Option<DestinationServiceQuery<T>>,
     dns_query: Option<IpAddrListFuture>,
     responders: Vec<Responder>,
+    tls_config_version: usize,
```

Comment:
Does the `DestinationSet` really need to store the config version? I think only the `Background` needs to store it